django/db/migrations/state.py

```diff
@@ -969,6 +970,7 @@ def render(self, apps):
             bases = tuple(
                 (apps.get_model(base) if isinstance(base, str) else base)
                 for base in self.bases
+                if base != typing.Generic
```

Comment:
Should we prefer `issubclass()` to avoid an exact check on this class (and therefore better support subclassing)?

I'm not speaking from experience here, but a little search shows there could be code like this? https://discuss.python.org/t/clarifying-the-rules-for-subclassing-generic-classes/69698

Reply:
Thanks for the review, I've added a test case with a more convoluted custom generic class taken [from the docs](https://typing.python.org/en/latest/spec/generics.html#arbitrary-generic-types-as-base-classes).

But It does not require additional changes I think because the `Cannot inherit from plain Generic` limitation is only for exact subclasses of plain `Generic` https://github.com/python/cpython/blob/4f18916c5c28321f363e85ee7ef3ee29bbf79d8e/Lib/typing.py#L1144.

```python
import typing

T = typing.TypeVar('T')

class MyGeneric(typing.Generic[T]): ...

  
class Ok(MyGeneric): ... # OK

class Broken(typing.Generic): ... # TypeError: Cannot inherit from plain Generic
```